requirements/requirements.txt

```diff
@@ -1,3 +1,4 @@
 motor==2.4.0
 pytest==6.2.4
 pytest-asyncio==0.15.1 # Needed for async pytest fixtures
+aiohttp==3.7.4.post0
```

Comment:
The way that I currently have `pytest-motor` set up is that the scripts use frozen requirements files in `requirements/frozen`. To update the frozen requirements files based on changes to the requirements files, you can run `scripts/freeze.sh`. Also, aiohttp should be added to `install_requires` in `setup.py`.